simple/templates/_container.tpl

```diff
@@ -34,9 +42,13 @@
         {{- end }}
         {{- if hasKey . "volumes" }}
         volumeMounts:
-          {{- range $key, $vol := $.volumes }}
-          - name: {{ $vol.name }}
-            mountPath: {{ $vol.path }}
+          {{- range $.volumes }}
```

Comment:
revert this change, using key-value pair for iteration is more preferred in helm templating